### PR TITLE
Jira auth

### PR DESCRIPTION
**Pull request by mattmurp**

### CODE REVIEW

**Comment by mattmurp** [comment]
This allows for the use of a personal access token for Jira authentication.

File: providers/jira/JIRAConstants.groovy [hunk 1]
```diff
@@ -27,9 +28,10 @@ public class JIRAConstants {
 	static def PROVIDER_DESCRIPTION =
 	[
 		'JIRA provider. Configuration fields are below',
-		'(Required)' + SERVER_URL    + ': the JIRA URL to connect to',
-		'(Required)' + USERNAME      + ': User name',
-		'(Required)' + PASSWORD      + ': Password',
+		'(Required)' + SERVER_URL    + ': The JIRA URL to connect to',
+		'(Optional)' + TOKEN         + ': A personal access token. Required if not using username and password.',
```
Comment (YuntaoCode) [inline providers/jira/JIRAConstants.groovy:32]: Do we need to change (Optional) to (Recommended) to be consistent with README.md?

Comment (mattmurp) [reply providers/jira/JIRAConstants.groovy]: Yeah, good point. Change made.

**Review by YuntaoCode — APPROVED** [review]
Looks good.

**Pull request merged**